### PR TITLE
Create car

### PR DESCRIPTION
**Pull request by gabrielcoder247**
## Create add new page item which will expect these information to create new car:

- url : required
- description : optional
- rent fee : required
- model year: required
- brand: required
- model: required

### CODE REVIEW

File: src/components/cars/newcar.js [hunk 1]
```diff
@@ -1,7 +1,3646 @@
...
+        className="btn btn-success btn-lg mt-5"
+        style={
+            { marginLeft: '35%' }
+        }
+      >
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
```
Comment (xtrahuman) [inline src/components/cars/newcar.js]: there is multiple empty spaces in your file. kindly removes these extra spaces to make the code concise

Also Kindly see how your project appears below.
image
desktop view

image
mobile view

Please make the form to appear in the center of the page

**Review by xtrahuman — CHANGES_REQUESTED** [review]
Hi @gabrielcoder247 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._ ⬇️  ⬇️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

File: src/components/cars/newcar.js (hunk 1 above)
Comment (gabrielcoder247) [reply src/components/cars/newcar.js]: I was all working fine perior to when I git pull from dev branch, there's probably a CSS clash, with my inline CSS, for the spaces that eslint, I will do without eslint so the task assigned will do that

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @gabrielcoder247 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

**Review by xtrahuman — APPROVED** [review]
Hello @gabrielcoder247 ,

Your project is complete! There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏

Congratulations! 🎉 🎉 🎉

Cheers and Happy coding!👏👏👏

**Pull request merged**